modules/bootconf.py

```diff
@@ -118,3 +127,94 @@ def configure(self, old, new):
             return False
 
         return True
+
+class UBootBeagleboneBootloader(BootloaderConfigurator):
+
+    def switchUEnv(self, old, new, uEnvPath):
+        ''' Switch boot partition using uEnv bootpart variable '''
```

Comment:
Does that mean that we have dual boot partitions as well as dual root partitions?